README.md

```diff
@@ -1,5 +1,13 @@
 [![github banner-short](https://user-images.githubusercontent.com/22394/26989908-dd99cc2c-4d22-11e7-9576-c6aeada2bd63.png)](http://luckyframework.org)
 
+![Deploy docs](https://github.com/luckyframework/lucky/workflows/Deploy%20docs/badge.svg)
+![Lucky CI](https://github.com/luckyframework/lucky/workflows/Lucky%20CI/badge.svg)
```

Comment:
I tend to think these aren't as useful to most people viewing the repo and sometimes they go red which makes people nervous even though it is only for master. So I think these 2 can be removed. The rest look great though!

Reply:
I think you're right. I'll revert the commit.